schema.yaml

```diff
@@ -11,179 +11,18 @@ properties:
       type: NAMESPACE
   glooctlInstaller.image.full:
     type: string
-    default: gcr.io/solo-io-public/gloo/installer:1.3
+    default: gcr.io/solo-io-public/gloo/installer:1.4.0
```

Comment:
I think this needs to match the app version as well

Reply:
Updated